client/rpcserver/handlers.go

```diff
@@ -62,7 +62,7 @@ func createResponse(op string, res interface{}, resErr *msgjson.Error) *msgjson.
 // *msgjson.ResponsePayload.
 func usage(route string, err error) *msgjson.ResponsePayload {
 	usage, _ := commandUsage(route, false)
-	err = fmt.Errorf("%v\n\n%s", err, usage)
+	err = fmt.Errorf("%w\n\n%s", err, usage)
 	resErr := msgjson.NewError(msgjson.RPCArgumentsError, err.Error())
```

Comment:
This `err` shouldn't even be an `error` from what I can tell, it's just used for the string.

```suggestion
	resErr := msgjson.NewError(msgjson.RPCArgumentsError, "%v\n\n%s", err, usage)
```